bindings/rust/evmc-declare/src/lib.rs

```diff
@@ -395,52 +395,34 @@ fn build_execute_fn(names: &VMNameSet) -> proc_macro2::TokenStream {
             host: *const ::evmc_vm::ffi::evmc_host_interface,
             context: *mut ::evmc_vm::ffi::evmc_host_context,
             revision: ::evmc_vm::ffi::evmc_revision,
-            msg: *const ::evmc_vm::ffi::evmc_message,
-            code: *const u8,
-            code_size: usize
+            msg: *const ::evmc_vm::ffi::evmc_message
         ) -> ::evmc_vm::ffi::evmc_result
         {
             use evmc_vm::EvmcVm;
 
-            // TODO: context is optional in case of the "precompiles" capability
```

Comment:
why is this removed?

Reply:
I wasn't able to translate the check easily and also not sure if this check isn't to heavy to be included in the bindings (unless this helps Rust translation somehow).

Reply:
Maybe `TODO` is worth to leave? It seems to point to something that is not yet implemented in bindings, but might be useful.

Reply:
I don't think the check for instance/msg null should be removed. This is the place it ensures it won't be null.